examples/gno.land/r/gnoland/valopers/init.gno

```diff
@@ -0,0 +1,72 @@
+package valopers
+
+import (
+	"gno.land/p/demo/avl"
+	"gno.land/p/moul/authz"
+)
+
+func init() {
+	valopers = avl.NewTree()
+
+	auth = authz.NewWithAuthority(
+		authz.NewContractAuthority(
+			"gno.land/r/gov/dao/v2",
+			func(title string, action authz.PrivilegedAction) error {
+				return action()
+			},
+		),
+	)
+
+	instructions = `
+# Welcome to the **Valopers** realm
+
+## 📌 Purpose of this Contract
+
+The **Valopers** contract is designed to maintain a registry of **validator profiles**. This registry provides essential information to **GovDAO members**, enabling them to make informed decisions when voting on the inclusion of new validators into the **valset**.
+
+By registering your validator profile, you contribute to a transparent and well-informed governance process within **gno.land**.
+
+---
+
+## 📝 How to Register Your Validator Node
+
+To add your validator node to the registry, use the **Register** function with the following parameters:
+
+- **Moniker**: A human-readable name for your validator.
+- **Description**: A detailed introduction in Markdown, including answers to the questions below.
+- **Validator Address**: The address of your validator node.
+- **Validator Public Key**: Your validator node’s public key.
+
+### ✍️ Required Information for the Description
+
+Please provide detailed answers to the following questions to ensure transparency and improve your chances of being accepted:
+
+1. The name of your validator
```

Comment:
Do we want the `Moniker` + the name in the description? (just checking but I'm fine with both options)

Reply:
We already have the moniker in the struct's field. I think it's good just to have the name in the description.